src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -70,9 +70,21 @@ public Long submitAppeal(SyaCaseWrapper appeal, String userToken) {
         return (caseDetails != null) ? caseDetails.getId() : null;
     }
 
-    public SaveCaseResult submitDraftAppeal(String oauth2Token, SyaCaseWrapper appeal) {
+    public Optional<SaveCaseResult> submitDraftAppeal(String oauth2Token, SyaCaseWrapper appeal) {
         appeal.setCaseType("draft");
-        return saveDraftCaseInCcd(convertSyaToCcdCaseData(appeal), getUserTokens(oauth2Token));
+        IdamTokens idamTokens = getUserTokens(oauth2Token);
+        try {
+            return Optional.of(saveDraftCaseInCcd(convertSyaToCcdCaseData(appeal), idamTokens));
+        } catch (FeignException e) {
+            if (e.status() == 409) {
```

Comment:
looks OK, Sunil. We could even check that the message of the error is the one we want to catch from CCD.